workbench-core/authorization/src/dynamicAuthorization/wbcGroupManagementPlugin.test.ts

```diff
@@ -326,4 +327,45 @@ describe('WBCGroupManagemntPlugin', () => {
       expect(mockUserManagementPlugin.addUserToRole).toBeCalledWith('userId', 'groupId');
     });
   });
+
+  describe('getGroupUsers', () => {
+    let groupId: string;
+    let userIds: string[];
+
+    beforeEach(() => {
+      groupId = 'userId';
+      userIds = ['123', '456', '789'];
+    });
+
+    it('returns an array of userID in the data object for the requested group', async () => {
+      mockUserManagementPlugin.listUsersForRole = jest.fn().mockResolvedValue(userIds);
+      const response = await wbcGroupManagementPlugin.getGroupUsers({ groupId, authenticatedUser: mockUser });
+
+      expect(response).toMatchObject<GetGroupUsersResponse>({ data: { userIds } });
+    });
+
+    it('throws IdpUnavailableError when the IdP encounters an error', async () => {
+      mockUserManagementPlugin.listUsersForRole = jest.fn().mockRejectedValue(new IdpUnavailableError());
+
+      await expect(
+        wbcGroupManagementPlugin.getGroupUsers({ groupId, authenticatedUser: mockUser })
+      ).rejects.toThrow(IdpUnavailableError);
+    });
+
+    it('throws PluginConfigurationError when the UserManagementService has a configuration error', async () => {
+      mockUserManagementPlugin.listUsersForRole = jest.fn().mockRejectedValue(new PluginConfigurationError());
+
+      await expect(
+        wbcGroupManagementPlugin.getGroupUsers({ groupId, authenticatedUser: mockUser })
+      ).rejects.toThrow(PluginConfigurationError);
+    });
+
+    it('throws GroupNotFoundError when the group doesnt exist', async () => {
+      mockUserManagementPlugin.listUsersForRole = jest.fn().mockRejectedValue(new GroupNotFoundError());
```

Comment:
user management should throw a `RoleNotFoundError`

Reply:
Fixed !